2025 Haunted House/src/main/java/frc/robot/subsystems/MeniMech.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class MeniMech extends SubsystemBase {
+  
+  private double handUpTime = 7;
+  private double handDownTime = 3;
+  private double headUpTime = 5;
+  private double headDownTime = 5;
+  private Timer headTimer;
+  private Timer handTimer;
```

Comment:
In Java you can declare multiple variables on one line: https://www.w3schools.com/java/java_variables_multiple.asp
You should do this here.

Reply:
fixed in [this commit](https://github.com/grt192/2024-Haunted-House/commit/dd91e1b7b2d58078fcbce140b21c609f1dc647bf)

Reply:
There's a few more places you could do this sort of consolidation.

For a later time: I notice the logic in the class is the same for `head` and `hand`, which indicates that it could be split into 2 instances of a class. I wouldn't work on that until all the other code is merged and working.